app/models/user.rb

```diff
@@ -36,6 +40,46 @@ def self.new_with_session(params, session)
     end
   end
 
+  def active_for_authentication?
+    super && check_if_account_active
+  end
+
+  def check_if_account_active
+    return true if Rails.env.test?
```

Comment:
after thinking about this a bit, I think that this should be removed here because we should have a test to make sure this test works. to make the tests work we can mock/stub `get_user_info` or move this line into that method.

Reply:
Maybe this would be good to pair over the IP sprint this part.     If I mock get_user_info, the check_if_account_active is called by lots of tests, where is the best place to mock this (without being redundant in code?)
(See the latest code, changed a bit)